.github/CONTRIBUTING.md

```diff
@@ -6,14 +6,19 @@ discussion.
 
 ## Development
 
-Check out this repo with Android Studio. It's a standard gradle project and conventional to checkout.
+Check out this repo with Android Studio. It's a standard gradle project and conventional to
+checkout.
+
+If you have build issues, you may need to run [kdoctor](https://github.com/Kotlin/kdoctor) to
+ensure you have all of the required dependencies, as this project uses KMP and therefore needs a few
+extra things installed beyond typical Android development such as Ruby and Cocoapods.
```

Comment:
Let's just link their docs rather than dupe

```suggestion
Circuit is a Kotlin Multiplatform project, so ensure you have your environment set up 
accordingly: https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-setup.html
```